dubbo-xds/src/test/java/org/apache/dubbo/registry/xds/util/bootstrap/BootstrapperTest.java

```diff
@@ -0,0 +1,124 @@
+            "      ],\n" +
+            "      \"POD_PORTS\": \"[{\\\"containerPort\\\":17070,\\\"protocol\\\":\\\"TCP\\\"},{\\\"containerPort\\\":17171,\\\"protocol\\\":\\\"TCP\\\"},{\\\"containerPort\\\":8080,\\\"protocol\\\":\\\"TCP\\\"},{\\\"name\\\":\\\"tcp-health-port\\\",\\\"containerPort\\\":3333,\\\"protocol\\\":\\\"TCP\\\"}]\",\n" +
+            "      \"PROV_CERT\": \"var/run/secrets/istio/root-cert.pem\",\n" +
+            "      \"PROXY_CONFIG\": {\n" +
+            "        \"binaryPath\": \"/usr/local/bin/envoy\",\n" +
+            "        \"concurrency\": 2,\n" +
+            "        \"configPath\": \"./etc/istio/proxy\",\n" +
+            "        \"controlPlaneAuthPolicy\": \"MUTUAL_TLS\",\n" +
+            "        \"discoveryAddress\": \"istiod.istio-system.svc:15012\",\n" +
+            "        \"drainDuration\": \"45s\",\n" +
+            "        \"holdApplicationUntilProxyStarts\": true,\n" +
+            "        \"parentShutdownDuration\": \"60s\",\n" +
+            "        \"proxyAdminPort\": 15000,\n" +
+            "        \"serviceCluster\": \"istio-proxy\",\n" +
+            "        \"statNameLength\": 189,\n" +
+            "        \"statusPort\": 15020,\n" +
+            "        \"terminationDrainDuration\": \"5s\",\n" +
+            "        \"tracing\": {\n" +
+            "          \"zipkin\": {\n" +
+            "            \"address\": \"zipkin.istio-system:9411\"\n" +
+            "          }\n" +
+            "        }\n" +
+            "      },\n" +
+            "      \"SERVICE_ACCOUNT\": \"default\",\n" +
+            "      \"WORKLOAD_NAME\": \"dubbo-demo-consumer-deployment-grpc-agent\"\n" +
+            "    },\n" +
+            "    \"locality\": {},\n" +
+            "    \"UserAgentVersionType\": null\n" +
+            "  },\n" +
+            "  \"certificate_providers\": {\n" +
+            "    \"default\": {\n" +
+            "      \"plugin_name\": \"file_watcher\",\n" +
+            "      \"config\": {\n" +
+            "        \"certificate_file\": \"/var/lib/istio/data/cert-chain.pem\",\n" +
+            "        \"private_key_file\": \"/var/lib/istio/data/key.pem\",\n" +
+            "        \"ca_certificate_file\": \"/var/lib/istio/data/root-cert.pem\",\n" +
+            "        \"refresh_interval\": \"900s\"\n" +
+            "      }\n" +
+            "    }\n" +
+            "  },\n" +
+            "  \"server_listener_resource_name_template\": \"xds.istio.io/grpc/lds/inbound/%s\"\n" +
+            "}";
+        BootstrapperImpl.bootstrapPathFromEnvVar = "";
+        BootstrapperImpl bootstrapper = new BootstrapperImpl();
+        bootstrapper.setFileReader(createFileReader(rawData));
+        Bootstrapper.BootstrapInfo info = bootstrapper.bootstrap();
+        List<Bootstrapper.ServerInfo> serverInfoList = info.servers();
+        MatcherAssert.assertThat(serverInfoList.get(0).target(), equalTo("unix:///etc/istio/proxy/XDS"));
```

Comment:
use Assertions.assertEquals

Reply:
Thank you, chengming, I have involved Junit.